webpack-demo/src/tasks.js

```diff
@@ -1,34 +1,22 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable no-plusplus */
 /* eslint-disable linebreak-style */
-/* eslint for-direction: "error" */
 export const saveTasks = (tasks) => {
   localStorage.setItem('tasks', JSON.stringify(tasks));
 };
 
-// Define the functions for adding tasks
 export const addTask = (description, tasks) => {
-  const task = {
-    description,
-    completed: false,
-    index: tasks.length + 1,
-  };
-  tasks.push(task);
+  const newTask = { description, completed: false };
+  tasks.push(newTask);
   saveTasks(tasks);
-  return task;
 };
```

Comment:
- [ ] Kindly consider adding the `index` property to the task object as it was a requirement in the first milestone of this object. In your project, the indexes of tasks only show up when the user clears some completed tasks by clicking on the `clear all completed` button, then the indexes of the remaining tasks show up.
The requirement is to save the indexes of tasks in the order in the local storage. 
so kindly consider fixing this bug to fulfill the requirement.
![qw](https://user-images.githubusercontent.com/96683843/230663758-0f08d1b1-8226-47af-883e-8f4f8118c671.jpg)